<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterUser</class>
 <widget class="QWizard" name="RegisterUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#cccccc;
</string>
  </property>
  <widget class="QWizardPage" name="userWizard">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string>Repeat Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmEmail">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPassword">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
border:1px solid #999999;
background-color: qlineargradient(spread:repeat, x1:0.486975, y1:0, x2:0.492462, y2:1, stop:0 rgba(144, 152, 149, 255), stop:0.19598 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QFrame" name="backgroundFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid white;
border-radius:18px;
background-color:white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>541</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>backgroundFrame</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lastName</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>firstName</zorder>
   <zorder>email</zorder>
   <zorder>confirmEmail</zorder>
   <zorder>password</zorder>
   <zorder>confirmPassword</zorder>
  </widget>
  <widget class="QWizardPage" name="userWizardFinish">
   <widget class="QFrame" name="backgroundFrame2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid white;
border-radius:18px;
background-color:white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="progressMessage">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>541</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>email</tabstop>
  <tabstop>confirmEmail</tabstop>
  <tabstop>password</tabstop>
  <tabstop>confirmPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
